configurations/admin.template.ini

```diff
@@ -163,6 +163,14 @@ moduls.kalturaLiveStreamTranscode.basePermissionType = 2
 moduls.kalturaLiveStreamTranscode.basePermissionName = FEATURE_KALTURA_LIVE_STREAM
 moduls.kalturaLiveStreamTranscode.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.liveStreamSegmentDuration.enabled = true
```

Comment:
must contain "Kaltura Live" at the beginning -> "Kaltura Live Dynamic segment duration support"